cmd/minishift/cmd/config/view.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 const (
-	DefaultConfigViewFormat = "- {{.ConfigKey | printf \"%-21s\"}}: {{.ConfigValue}}"
+	DefaultConfigViewFormat = "- {{.ConfigKey | printf \"%-35s\"}}: {{.ConfigValue}}"
```

Comment:
will this not just postpone the issue until longer strings are used?

Reply:
@gbraad thats, right. But I do not see any better alternative. Also it is very hard to support the `--format` with tabwriter. So the question is is it really required to put a lot of effort for this. IMO this simple fix which will be enough for us and there is `--format` anyway if someone wants the format in specific way.

Reply:
@gbraad I am going to merge it, If you have other suggestions please open an new issue.